uploadAlignments.m

```diff
@@ -18,7 +18,7 @@
 chrom = fetchn(seq.AssemblyUnit & 'assembly="mm10"','assembly_unit');
```

Comment:
Of course, we should replace all upload scripts with automatic population of tables.  This ensures that all data are complete.